src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -888,16 +888,16 @@ public function set_status( $status ) {
 		}
 
 
-		$status = wp_update_post( array(
-			'ID' => $this->id,
-			'post_status' => Tribe__Events__Aggregator__Records::$status->{ $status },
-		) );
+		$status = wp_update_post( [
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: d5bceefd3155436abc347f9ce1d9f376</sub>